pypeit/calibrations.py

```diff
@@ -58,6 +58,8 @@ class Calibrations:
         show (:obj:`bool`, optional):
             Show plots of PypeIt's results as the code progesses.
             Requires interaction from the users.
+        slitspat_num (??):
```

Comment:
I think this allows one to limit reductions to a single slit.
Or it did..

Reply:
If anyone can provide some text for this, I'll add it.

Reply:
It's fine to leave as `??` for now.